ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml

```diff
@@ -829,6 +829,50 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: baremetal-compact-agent-ipv4-static-connected-f7-compliance
```

Comment:
This is a workflow without default sc created, all the test cases will be skipped on it.